open_instruct/finetune.py

```diff
@@ -219,10 +219,10 @@ def main(args: FlatArguments):
             args.dataset_name,
             args.dataset_config_name,
         )
-    elif args.dataset_mixer is not None:
+    elif args.dataset_mixer_list is not None:
```

Comment:
This looks weird. Isn't this going to fail with the existing configs where we just pass in the dataset mixer and not the list object?

Reply:
I realized I can just check if either is not none (b/c we enforce that only one can be not none at a time), so now this should be able to handle both cases